ts/utils/__tests__/url.test.ts

```diff
@@ -57,3 +65,26 @@ describe("getUrlBasepath", () => {
     });
   });
 });
+
+describe("extractPathFromURL", () => {
+  it("should return undefined", () => {
+    const url = "https://www.google.com/it/hello";
+    expect(
+      extractPathFromURL(
+        [IO_INTERNAL_LINK_PREFIX, IO_UNIVERSAL_LINK_PREFIX],
+        url
+      )
+    ).toBeUndefined();
+  });
+
+  it("should return the path", () => {
+    const url = "https://continua.io.pagopa.it/idpay/auth/12345678";
+    const path = "/idpay/auth/12345678";
```

Comment:
Should be possibile to add a test with an url like this "https://continua.io.pagopa.it/initiative/main?itemId=12345678" ?